pkgs/top-level/aliases.nix

```diff
@@ -853,6 +853,10 @@ mapAliases ({
   vivaldi-widevine = throw "'vivaldi-widevine' has been renamed to/replaced by 'widevine-cdm'"; # Added 2023-02-25
   vkBasalt = vkbasalt; # Added 2022-11-22
   vte_290 = throw "'vte_290' has been renamed to/replaced by 'vte'"; # Added 2023-01-05
+  varnish72 = throw "varnish 7.2 is EOL. Either use the LTS or upgrade."; # Added 2023-10-09
+  varnish73 = throw "varnish 7.3 is EOL. Either use the LTS or upgrade."; # Added 2023-10-09
```

Comment:
`varnish74` isn't in nixpkgs yet, so you currently can't upgrade.

I can add it. Would you prefer me to just push that to this branch, open a PR against your branch, open a new PR against nixpkgs, … ?